feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+
+	tcArgs.client.Modify().AddEntry(t,
+		fluent.IPv4Entry().
+			WithPrefix(pfx4AdvV4.ip+"/24").
+			WithNetworkInstance(deviations.DefaultNetworkInstance(dut)).
+			WithNextHopGroup(nhgID1))
+}
+
+// startTraffic start traffic on ATE
+func startTraffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, top gosnappi.Config) {
+	t.Helper()
+	capturePktsBeforeTraffic(t, dut, dutPortList)
+	time.Sleep(10 * time.Second)
+	ate.OTG().StartTraffic(t)
+	time.Sleep(time.Minute)
+	ate.OTG().StopTraffic(t)
+	time.Sleep(time.Second * 40)
+	otgutils.LogFlowMetrics(t, ate.OTG(), top)
+	otgutils.LogLAGMetrics(t, ate.OTG(), top)
+	otgutils.LogPortMetrics(t, ate.OTG(), top)
+}
+
+// capturePktsBeforeTraffic capture the pkts before traffic on DUT Ports
+func capturePktsBeforeTraffic(t *testing.T, dut *ondatra.DUTDevice, dutPortList []*ondatra.Port) {
+	rxPktsBeforeTraffic = map[*ondatra.Port]uint64{}
+	txPktsBeforeTraffic = map[*ondatra.Port]uint64{}
+	for _, port := range dutPortList {
+		rxPktsBeforeTraffic[port] = gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().InPkts().State())
+		txPktsBeforeTraffic[port] = gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().OutPkts().State())
+	}
+}
+
+// verifyTrafficFlow verify the each flow on ATE
+func verifyTrafficFlow(t *testing.T, ate *ondatra.ATEDevice, flows []gosnappi.Flow) bool {
+	if flows[0].Name() == "pfx1ToPfx4" {
+		rxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flows[0].Name()).Counters().InPkts().State())
+		if got := rxPkts / 100; got == 0 {
+			t.Logf("Packet Dropped, LossPct for flow %s: got %d, want 0 packet", flows[0].Name(), got)
+		} else {
+			return false
+		}
+	} else {
+		for _, flow := range flows {
+			rxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flow.Name()).Counters().InPkts().State())
+			txPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flow.Name()).Counters().OutPkts().State())
+			lostPkt := txPkts - rxPkts
+			if got := (lostPkt * 100 / txPkts); got > 0 {
+				return false
+			}
+		}
+	}
+	return true
+}
+
+// awaitAdjacency wait for adjacency to be up/down
+func awaitAdjacency(t *testing.T, dut *ondatra.DUTDevice, intfName string, state oc.E_Isis_IsisInterfaceAdjState) bool {
+	isisPath := ocpath.Root().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance).Isis()
+	intf := isisPath.Interface(intfName)
+	query := intf.LevelAny().AdjacencyAny().AdjacencyState().State()
+	_, ok := gnmi.WatchAll(t, dut, query, time.Minute, func(val *ygnmi.Value[oc.E_Isis_IsisInterfaceAdjState]) bool {
+		v, ok := val.Val()
+		return v == state && ok
+	}).Await(t)
+
+	return ok
+
+}
+
+// checkBidirectionalTraffic verify the bidirectional traffic on DUT ports.
+func checkBidirectionalTraffic(t *testing.T, dut *ondatra.DUTDevice, portList []*ondatra.Port) error {
+
+	for _, port := range portList {
+		txPkts := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().InPkts().State())
```

Comment:
There is typo , change InPkts() to Outpkts()

Reply:
Thanks @bkreddy143 for finding this. Changes are done.